environment.yml

```diff
@@ -0,0 +1,43 @@
+name: plateau
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - dask[dataframe]!=2021.5.1,!=2021.6.0  # gh475 - 2021.5.1 and 2021.6.0 broke ci, omit those versions
```

Comment:
Does the `[dataframe]` syntax do anything?

Reply:
No, it doesn't.